tests/publiccloud/storage_perf.pm

```diff
@@ -120,6 +154,33 @@ sub run {
                 values => $values
             };
             $data = influxdb_push_data($url, 'publiccloud', $data);
+            my %influx_read_args = (
+                url => $url,
+                db => 'publiccloud',
+                scenario => $href->{name},
+                os_flavor => $tags->{os_flavor},
+                os_version => $tags->{os_version}
+            );
+            if (db_has_data(%influx_read_args, timeframe => $this_week_timeframe) && db_has_data(%influx_read_args, timeframe => $previous_week_timeframe)) {
+                foreach my $load_type (keys %$values) {
+                    $influx_read_args{load_type} = $load_type;
+                    my $current_load_type = sprintf(" load_type=%s, scenario=%s, Flavor=%s, Version=%s", $influx_read_args{load_type}, $influx_read_args{scenario}, $influx_read_args{os_flavor}, $influx_read_args{os_version});
+                    my $this_week = get_median_from_db(%influx_read_args, timeframe => $this_week_timeframe);
+                    my $previous_week = get_median_from_db(%influx_read_args, timeframe => $previous_week_timeframe);
+                    if (defined($previous_week) && defined($this_week) && $previous_week != 0 && $this_week != 0) {
+                        record_info('ANALYZE', "Checking " . $current_load_type);
```

Comment:
```suggestion
                        record_info('ANALYZE', "Checking " . $current_load_type);
                        record_info('This week', "Median for this week: " . $this_week);
                        record_info('Previous week', "Median for previous week: " . $previous_week);
```

Reply:
I can not fully follow your suggestions due to some changes . But I think I follow them as much as I can :)